doc/install.rst

```diff
@@ -33,87 +49,135 @@ scikit-image >= 0.12.3, a few examples require pandas >= 0.18.0.
     Scikit-learn now requires Python 3.5 or newer.
 
 If you already have a working installation of numpy and scipy,
-the easiest way to install scikit-learn is using ``pip`` ::
+the easiest way to install scikit-learn is using ``conda`` (see the
```

Comment:
I think `pip` should still be described before `conda`.

Installing `conda` requires a 2GB+ download, it's really not negligible

Reply:
Done